test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -168,11 +166,14 @@ private static void createAndShowGUI() {
         radioBtnSingle = new JRadioButton("Not Grouped");
         radioBtnSingle.setSelected(true);
 
-        mainFrame.getContentPane().add(btnStart);
-        mainFrame.getContentPane().add(box);
-        mainFrame.getContentPane().add(radioBtnSingle);
-        mainFrame.getContentPane().add(btnEnd);
+        // Create main vertical box
+        Box mainBox = Box.createVerticalBox();
```

Comment:
```suggestion
        Box mainBox = Box.createVerticalBox();
```
The comment is redundant, it doesn't add anything to what one could infer from the statement.